config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-end
+  root 'pages#index'
+  get '/pages' => 'pages#index'
+  get 'pages/meetup' => 'pages#meetup'
+  get 'pages/contact' => 'pages#contact'
+  get 'pages/about' => 'pages#about'
+end
```

Comment:
Nothing wrong with this code, I just want to let you know that instead of:
`get 'pages/contact' => 'pages#contact'`
you can also do:
`get 'pages/contact, to: 'pages#contact'`

I have an irrational hatred of hash rockets so that just my be my issues coming through :)

Reply:
OOOoo I like it